src/main/java/org/iac2/repository/compliancejob/TriggerRepository.java

```diff
@@ -1,12 +1,17 @@
 package org.iac2.repository.compliancejob;
 
+import java.util.List;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
 import org.iac2.entity.compliancejob.trigger.TriggerEntity;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.rest.core.annotation.RestResource;
 
-import java.util.List;
-
+@RepositoryRestResource(path = "triggers")
+@Tag(name = "trigger")
 public interface TriggerRepository extends CrudRepository<TriggerEntity, Long> {
     @Query("""
```

Comment:
we really need a query here? ByJobId is not working?

Reply:
The relation is ManyToMany between triggers and jobs. That's why a query was necessary